osler/inventory/forms.py

```diff
@@ -0,0 +1,43 @@
+from django import forms
+from django.forms import ModelForm, Form
+from . import models
+
```

Comment:
Make sure you clean up all your imports on all your files, you import and bunch of things you don't use